.github/workflows/dockerimages.yml

```diff
@@ -0,0 +1,35 @@
+name: Docker Images CI
+
+on: [pull_request]
+
+env:
+  PLATFORM: linux/amd64
+
+jobs:
+  build:
+    name: Build image
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        file: [Dockerfile, Dockerfile.slim]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          version: latest
+
+      - name: Build the Docker image
+        run: |
+          docker buildx build \
+            --tag renovate \
+            --file ./${{ matrix.file }} .
+
+      - name: Image history
+        run: docker history renovate
```

Comment:
Any command to easily show us the *total* size? e.g. `docker images | grep renovate` ?